style.css

```diff
@@ -1 +1,254 @@
+}
+
+/* each box is width=201 and heith=125 */
+
+img {
+    width: 100%;
+    height: 100%;
+}
+
+.footer-section {
+    display: flex;
+    flex-direction: column;
+    gap: 4rem;
+    height: auto;
+    
+}
+.side-footer {
+    display: flex;
+    flex-direction: column;
+    height: 7rem;
+    width: 14rem;
+    margin: 2rem 1rem 1rem 1rem;
+}
+
+.side-footer-item {
+    margin: 0.5rem;
+    background-color: #CDF3AC;
+    font-size: large;
+    font-weight: bold;
+    padding: 0.25em;
+}
+
+.center-footer {
+    align-items: end;
+    margin-left: auto;
+    margin-right: auto;
+    background-color: #F1D8D5;
+    margin-bottom: 2rem;
+    padding: 0.25rem;
+}
+
+
+@media screen and (min-width: 540px) {
+    body {
+        background-color: #d7e4ca;
+    }
+    .center-footer {
+    background-color: #d7e4ca;
+}
+}
+
+@media screen and (min-width: 900px) {
+    body {
+        background-color: #f3e3bc;
+    }
+    .cake-display {
+        grid-template-columns: 1fr 1fr 1fr 1fr;
+        grid-template-rows: auto;
+    }
+    .hamburger-menu {
+        display: none;
+    }
+
+    .large-nav {
+        display: flex;
+        flex-direction: row;
+        justify-content: right;
+        background-color: #F4ACB7;
+        height: auto;
+        width: auto;
+        margin-top: 2rem;
+        margin-bottom: 3rem;
+        padding: 0.25em 1em;
+    }
+    
+    .large-nav-item {
+        text-align: center;
+        font-size: large;
+        font-weight: bold;
+        background-color: #CDF3AC;
+        margin: 1em;
+        justify-content: space-evenly;
+        font-weight: bold;
+        padding: 0.5em;
+        width: 9rem;
+
+    }
+    
+
+    .slogan {        
+        display: flex;
+        background-color: #F4ACB7;
+        font-size: x-large;
+        font-weight: bold;
+        text-align: right;
+        justify-content: end;
+        align-items: end;
+        margin: 0 2em;
+        width: auto;
+    }
+
+    .desktop-hero {
+        display: flex;
+        flex-direction: row-reverse;
+        gap: 1rem;
+    }
+
+    /* .desktop-hero {
+        display: grid-reverse;
+        gap: 1rem;
+        grid-template-columns: 1fr 1fr;
+        grid-template-rows: auto;
+    } */
+
+    .footer-section {
+        display: flex;
+        flex-direction: column;
+        
+    }
+    .side-footer {
+        display: flex;
+        flex-direction: row;
+        height: auto;
+        width: auto;
+        margin-left: auto;
+        margin-right: auto;
+    }
+    
+    .side-footer-item {
+        margin: 6rem 2rem 0 2rem;
+        background-color: #CDF3AC;
+        font-size: x-large;
+        font-weight: bold;
+        padding: 0.25rem;
+        text-justify: center;
+        text-align: center;
+        width: 15rem;
+    }
+
+    .center-footer {
+        display: block;
+        margin-left: auto;
+        margin-right: auto;
+        background-color: #f3e3bc;
+        margin-bottom: 5rem;
+        width: 30rem;
```

Comment:
do we really want to use 30rem as the width? Ant reason why we aren't using a percentage?